docs/INSTALL.md

```diff
@@ -84,9 +84,8 @@ Installing *ccache* can speed up active development.
 _Ubuntu 20.04, 22.04:_
```

Comment:
Let's leave only 22.04 here, and also leave cmake installed in the command below.